### ISSUE TITLE
setting the source for search module

### ISSUE THREAD

**Issue by hyusetiawan** [Semantic-Org/Semantic-UI#4387]
i am using Search in conjunction with React, so the source will change dynamically but also locally.
After perusing the docs, I was hoping that there is something like .search('set source') but alas I didn't find it. Is there a way to update the source then?

**Comment by jlukic** [comment]
Yes, you can use the `setting` behavior to update the settings object without causing re-initialization (aka hot swapping settings).

```javascript
$('.ui.search').search('setting', 'source', newSource);
```

**Comment by hyusetiawan** [comment]
I just tried that and it didn't work.
I tried it locally and also, I tried changing the source from the [example page of source] (http://semantic-ui.com/modules/search.html) by:

1. select the country search input (I added an id to the country search)
2.  I tried changing the source like so: `$('#country').search('setting', 'source', [{ title: 'United Arab Emirates' },
  { title: 'Afghanistan' },
  { title: 'Antigua' }])`
3. the search input still searches for all countries, I expected it would only search the three countries I listed above
